### PR TITLE
Add linter config and update dependencies

### PR DESCRIPTION
**Pull request by mariapotashnyk**
Story: https://github.com/CyberCraftInc/cyber-dashboard/issues/87

Due to lack of react.js files, temporary eslint checks for errors all in .js files in app/javascript/ directory. It can be changed easily when react files will be added.

Example:

ESLINT

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,12513 @@
+{
```
Comment (alazarchuk) [inline package-lock.json]: Yarn complains on this file. Do we need it?

File: package.json [hunk 2]
```diff
@@ -2,10 +2,11 @@
   "name": "cyberdashboard",
   "private": true,
   "dependencies": {
-    "@rails/webpacker": "3.5",
+    "@rails/webpacker": "^4.0.7",
```
Comment (alazarchuk) [inline package.json]: What is the reason of upgrading this library?

File: package.json [hunk 3]
```diff
@@ -2,10 +2,11 @@
...
     "axios": "0.19.0",
     "bootstrap": "4.3.1",
     "imask": "4.1.4",
+    "jquery": "^3.4.0",
```
Comment (alazarchuk) [inline package.json]: We already have jquery?

File: package.json [hunk 4]
```diff
@@ -16,6 +17,11 @@
     "vue-template-compiler": "2.5.21"
   },
   "devDependencies": {
-    "webpack-dev-server": "2.11.2"
+    "eslint": "^6.6.0",
+    "eslint-plugin-react": "^7.16.0",
+    "webpack-dev-server": "^3.9.0"
```
Comment (alazarchuk) [inline package.json]: Same here. Do we need to upgrade this library?

**Comment by mariapotashnyk** [comment]
Replaced by https://github.com/CyberCraftInc/cyber-dashboard/pull/146 (used `yarn` instead of `npm`)

**Pull request closed without merging**